two-factor.php

```diff
@@ -151,7 +168,7 @@ function wpcom_enable_two_factor_plugin() {
 		return;	
 	}
 
-	wpcom_vip_load_plugin( 'two-factor' );
+	require_once( 'two-factor' );
```

Comment:
This will fatal :)

Should probably be:

```suggestion
	require_once( WPMU_PLUGIN_DIR . '/shared-plugins/two-factor/two-factor.php' );
```